chart/values.yaml

```diff
@@ -245,7 +245,7 @@ secret: []
 # in this case disable setting of those variables by setting the relevant configuration to false.
 enableBuiltInSecretEnvVars:
   AIRFLOW__CORE__FERNET_KEY: true
-  AIRFLOW__CORE__SQL_ALCHEMY_CONN: true
+  AIRFLOW__DATABASE__SQL_ALCHEMY_CONN: true
```

Comment:
Can you add backward compatiblty? We want to support all Airflow 2.0+ version in Helm Chart.
https://github.com/apache/airflow/blob/45a352e41692e22b7d161dc9b0c85bbf228d8f56/chart/values.yaml#L1443